charts/tools/values.yaml

```diff
@@ -0,0 +1,26 @@
+fullnameOverride: crunchy-postgres-tools
```

Comment:
Is this the example values.yaml file that defines the secret values file?

Reply:
Yes it is, do you think I should use a more generic name for the default?

Reply:
I guess it just wasn't clear to me that this was an example file. A comment at the top to make this clearer would help. And maybe an example for the networking.route.host value

Reply:
I'm a bit confused to be honest, this is the `values.yaml` that defines all of the  expected values. Isn't that standard with helm charts?